glean-core/android/src/main/java/mozilla/telemetry/glean/scheduler/PingUploadWorker.kt

```diff
@@ -147,7 +148,13 @@ class PingUploadWorker(context: Context, params: WorkerParameters) : Worker(cont
             WorkManager.getInstance(context).enqueueUniqueWork(
                 PING_WORKER_TAG,
                 ExistingWorkPolicy.KEEP,
-                buildWorkRequest<PingUploadWorker>(PING_WORKER_TAG))
+                buildWorkRequest<PingUploadWorker>(PING_WORKER_TAG)
+            )
+
+            // TODO: only do th9is in test mode
```

Comment:
typo.
Will this todo be fixed here or in a follow-up? If so, it should get a bug filed.

Reply:
This TODO was already fixed, sorry, it was a stray comment. `isSendingToTestEndpoint` is already only set when using the Glean test rule.